src/AstTranslator.h

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include "AstAnalyses.h"
```

Comment:
I think a forward declaration will do, with an include in the cpp file.

Reply:
I don't think so - this is a templatized class.